packages/dart_cli/bin/docs_page.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:args/args.dart';
+import 'dart:io';
+import 'package:path/path.dart' as path;
+import 'package:ansi_styles/extension.dart';
+
+void main(List<String> arguments) async {
+  final parser = ArgParser();
+  parser.addCommand('init');
+  final argResults = parser.parse(arguments);
+  if (argResults.command?.name == 'init') {
+    await createFiles();
+  } else {
+    throw UnsupportedError('Only the init command is currently supported.');
+  }
+}
+
+createFiles() async {
+  final indexContent = '# Welcome to your new documentation!';
+  final configContent = '''
+---
+name: ''
+logo: ''
+logoDark: ''
+favicon: ''
+socialPreview: ''
+twitter: ''
+noindex: false
+theme: "#00bcd4"
+sidebar: []
+headerDepth: 3
+variables: {}
+googleTagManager: ''
+zoomImages: false
+experimentalCodehike: false
+experimentalMath: false
+''';
+
+  var current = Directory.current;
+
+  var configPath = path.joinAll([current.path, 'docs.yaml']);
```

Comment:
```suggestion
  final configPath = path.joinAll([current.path, 'docs.yaml']);
```

Use final instead of var for variables that are not reassigned (like `const` vs `let` in JS)